File: repos_estudiantes_TARQUI_tercera_entrega/SIÑANI_QUELALI_IVAN/tarea3/per3java/Cliente.java
import java.io.Serializable;

public class Cliente implements Serializable {
    private int id;
    private String nombre;
    private int telefono;
    // Constructor
    public Cliente(int id, String nombre, int telefono) {
        this.id = id;
        this.nombre = nombre;
        this.telefono = telefono;
    }
    //Getters
    public int getId() { return id; }
    public String getNombre() { return nombre; }
    public int getTelefono() { return telefono; }
    //cliente
    @Override
    public String toString() {
        return "Cliente: " + nombre + " | ID: " + id + " | Teléfono: " + telefono;
    }
}

File: repos_estudiantes_TARQUI_tercera_entrega/SIÑANI_QUELALI_IVAN/tarea3/per3python/cliente.py
class Cliente:
    def __init__(self, id_cliente, nombre, telefono):
        self.id_cliente = id_cliente
        self.nombre = nombre
        self.telefono = telefono

    def __str__(self):
        return f"Cliente: {self.nombre} | ID: {self.id_cliente} | Teléfono: {self.telefono}"
